angio-frontend/src/modules/analyse/components/AnalyseListFilter.vue

```diff
@@ -0,0 +1,149 @@
+                                    v-bind:label="$t('analyse.component.analyseListFilter.starred')"
+                                    class="justify-end"
+                            ></v-checkbox>
+                        </v-flex>
+                    </v-layout>
+                </v-flex>
+            </v-layout>
+        </v-card-title>
+    </v-card>
+</template>
+
+<script lang="ts">
+    import {Component, Emit, Prop, Vue} from "vue-property-decorator";
+    import {AnalyseFilterModel, AnalyseStatusRepresentation, AnalyseStatusType} from "@/modules/analyse/models/analyse";
+    import {Locale} from "@/modules/user/store/userState";
+    import {AnalyseEvent} from "@/modules/analyse/AnalyseEvent";
+    import {debounce} from "helpful-decorators";
+    import PeriodDatePicker from '@/modules/common/components/PeriodDatePicker.vue';
+    import {PeriodDateModel} from '@/modules/common/models/PeriodDateModel';
+
+    @Component({
+        components: {PeriodDatePicker},
+    })
+    export default class AnalyseListFilter extends Vue {
+
+        @Prop()
+        public readonly locale!: Locale;
+
+        public search: string = '';
+
+        public statuses: AnalyseStatusRepresentation[] = [
+            { value: AnalyseStatusType.CREATED, text: 'analyse.component.analyseListFilter.status.CREATED' },
+            { value: AnalyseStatusType.SUCCESS, text: 'analyse.component.analyseListFilter.status.SUCCESS' },
+            { value: AnalyseStatusType.IN_PROGRESS, text: 'analyse.component.analyseListFilter.status.IN_PROGRESS' },
+            { value: AnalyseStatusType.FAILED, text: 'analyse.component.analyseListFilter.status.FAILED' }
+        ];
+
+        public selectedStatuses = [];
+
+        public isPeriod: boolean = true;
+
+        public isStarred: boolean = false;
+
+        public analyseFilterModel: AnalyseFilterModel = {
+            search: '',
+            statuses: [],
+            startDate: '',
+            endDate: '',
+            isStarred: false
+        };
+
+        public onPeriodDataChanged(periodDateModel: PeriodDateModel) {
+            if (periodDateModel.isPeriod) {
+                this.analyseFilterModel.startDate = periodDateModel.startDate;
+                this.analyseFilterModel.endDate = periodDateModel.endDate;
+            } else {
+                this.analyseFilterModel.startDate = periodDateModel.date;
+                this.analyseFilterModel.endDate = periodDateModel.date;
+            }
+        }
+
+        @debounce(1500)
+        @Emit(AnalyseEvent.SEND_ANALYSE_FILTER)
+        public submit() {
+            this.analyseFilterModel.search = this.search;
+            this.analyseFilterModel.statuses = this.selectedStatuses.map(status => (status as AnalyseStatusRepresentation).value);
+            this.analyseFilterModel.isStarred = this.isStarred;
+
+            return this.analyseFilterModel;
+        }
+    }
+</script>
+
+<style scoped>
```

Comment:
If component has no styles no need to define `<style></style>` block

Reply:
Done